.github/workflows/gate_fedora.yml

```diff
@@ -47,7 +47,9 @@ jobs:
                         rhel10 \
                         rhv4 \
                         uos20 \
-                        ocp4
+                        ocp4 \
+                        uos20 \
+                        al2023
```

Comment:
needs to be sorted alphabetically and is including twice the `uos20`

Reply:
Done - Thanks!